dch/sampleData.js

```diff
@@ -0,0 +1,33 @@
+
+"use strict"
+//Import database stuff: connection, 
+var required = require('./DB.js');
+const accelrationData = required.model;
+
+for(var i = 0; i < 10; i++){
+
+	for(var l = 0; l < 100; l++){
+			let x = {watchId: 'watch ' + i, X_ACCELERATION: (Math.random() * 100), Y_ACCELERATION:(Math.random() * 100), 
+			Z_ACCELERATION:(Math.random() * 100), 
+			TIMESTAMP: randomDate(new Date(2012,1,1,0,0,0,0), new Date(2016,1,1,0,0,0,0))}; 
+
```

Comment:
This is incorrect and deviates from the schema. This is actually a bug I introduced in the 'mqtt-subscriber.js' file. When I used X_ACCELERATION I meant for it to write out the string 'x_acc'  (the value X_ACCELERATION was set to) but I forgot that is inappropriate javascript syntax so it produced a JSON with field X_ACCELERATION.

The JSON should be:

```
var sampleData = {
  watch_id: 'watch ' + i,
  x_acc: (Math.random() * 100),
  y_acc: (Math.random() * 100), 
  z_acc: (Math.random() * 100), 
  timestamp: randomDate(new Date(2012,1,1,0,0,0,0), new Date(2016,1,1,0,0,0,0))
};
```

Also rename it from 'x'. Way too generic.